src/api/java/CMakeLists.txt

```diff
@@ -9,20 +9,59 @@
 ## directory for licensing information.
 ##
 
-# create a directory for the cvc package
-file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/cvc")
+# copy cvc libraries to the resource directory
+file(GLOB CVC_LIBS "${CMAKE_INSTALL_PREFIX}/lib/*.so*")
```

Comment:
I'd not recommend using glob here, it should be possible to get the location of `libcvc4` via the generator expressions `$<TARGET_FILE:cvc4>`

Reply:
Done